tikv: update the Load Base Split introduction (#9656)
pingcap/docs: configure-load-base-split.md

## Implementation principles

> Load Base Split automatically splits the Region based on statistics. It identifies the Regions whose read load or CUP usage consistently exceeds the threshold for 10 seconds, and splits these Regions at a proper position. When choosing the split position, Load Base Split tries to balance the access load of both Regions after the split and avoid access across Regions.

Reviewer: ```suggestion
Load Base Split automatically splits the Region based on statistics. It identifies the Regions whose read load or CPU usage consistently exceeds the threshold for 10 seconds, and splits these Regions at a proper position. When choosing the split position, Load Base Split tries to balance the access load of both Regions after the split and avoid access across Regions.
```

The Region split by Load Base Split will not be merged quickly. On the one hand, PD's `MergeChecker` skips the hot Regions; on the other hand, PD also determines whether to merge two Regions according to `QPS` in the heartbeat information, to avoid the merging of two Regions with high `QPS`.

## Usage

> The Load Base Split feature is currently controlled by three parameters:

Reviewer 2: ```suggestion
The Load Base Split feature is currently controlled by the following parameters:
```

> - `split.qps-threshold`: The threshold of QPS for the Region to be split, default is `3000` per second.

Reviewer 2: ```suggestion
- `split.qps-threshold`: The QPS threshold at which a Region is identified as a hotspot. The default value is `3000` per second.
```

> - `split.byte-threshold`: The threshold of read load for the Region to be split, default is 30 MiB per second.

Reviewer 2: ```suggestion
- `split.byte-threshold`: The traffic threshold at which a Region is identified as a hotspot. The default value is 30 MiB per second.
```

> - `split.region-cpu-overload-threshold-ratio`: The threshold of CPU usage (the percentage of CPU time of the read thread pool) for the Region to be split, default is `0.25`.

Reviewer 2: English and Chinese are inconsistent. Please check and update.

> If the sum of all types of read requests per second for a Region exceeds the QPS threshold or traffic threshold for 10 consecutive seconds, PD splits the Region.

Reviewer: What about the CPU usage threshold?

Reviewer: Also, it's not that "PD splits the Region", this is a TiKV feature that doesn't involve PD, so the precise description should be like "TiKV tries to split the Region".

> If the sum of all types of read requests per second for a Region exceeds the QPS threshold, traffic threshold, or CPU usage threshold for 10 consecutive seconds, TiKV tries to split the Region.

Reviewer 2: ```suggestion
If a Region meets one of the following conditions for 10 consecutive seconds, TiKV tries to split the Region:

- the sum of its read requests exceeds `split.qps-threshold`.
- its traffic exceeds `split.byte-threshold`.
- its CPU usage in the Unified Read Pool exceeds `split.region-cpu-overload-threshold-ratio`.
```
